decimalSuite_test.go

```diff
@@ -173,27 +219,27 @@ func runTest(testVals decValContainer, testValStrings testCaseStrings) (Decimal6
 
 // TODO: get runTest to run more functions such as FMA.
 // execOp returns the calculated answer to the operation as Decimal64.
-func execOp(val1, val2, val3 Decimal64, op string) Decimal64 {
+func execOp(context DecContext, testVals decValContainer, op string) Decimal64 {
```

Comment:
Is this parameter change warranted? `val1, val2, val3 Decimal64` seems fine to me.